packages/terra-date-time-picker/docs/README.md

```diff
@@ -0,0 +1,49 @@
+# Terra Date Time Picker
+
+The DateTimePicker component has a date picker for selecting date and a time input for entering time. DateTimePicker supports the Spring and Fall daylight saving time changes in the following behaviors:
+
+Spring: If you enter a date and time that falls within the missing hour during the Spring time change, the time will automatically be adjusted to the next relevant hour when the component loses focus. In the central time zone, the missing hour occurs between 2:00 and 2:59. For example, if the entered time is 2:15, the component will automatically adjust the time to the next hour at 3:15.
+
+Fall: If you enter a date and time that falls within the ambiguous hour during the Fall time change, the component will display a dialog to require you to select daylight or standard time. After a selection is made a button will be displayed with a label showing the selected option.
+
+
```

Comment:
Do we want to indicate this component is composed of the date-picker and time-input components?

Reply:
Updated: https://github.com/cerner/terra-core/commit/7f704a5dd8211fc1627f14d09ab2fa2b9f92f5ec